packages/tasks/src/local-apps.ts

```diff
@@ -103,6 +103,14 @@ export const LOCAL_APPS = {
 		displayOnModelPage: isGgufModel,
 		deeplink: (model) => new URL(`https://backyard.ai/hf/model/${model.id}`),
 	},
+	recursechat: {
+		prettyLabel: "RecurseChat",
+		docsUrl: "https://recurse.chat",
+		mainTask: "text-generation",
+		macOSOnly: true,
+		displayOnModelPage: isGgufModel,
+		deeplink: (model) => new URL(`recursechat://new-hf-gguf-model?hf-model-id=${model.id}`),
```

Comment:
hi @julien-c Thanks for the heads up! We'll look into adding it, but it'll take some time to release it.

I've updated the PR branch. Would it be fine to add this as is and we'll make another PR for file path url later? It would help largely simplify our current users experience to add a model from HF.